frontend/package.json

```diff
@@ -14,7 +14,8 @@
     "build": "ember build",
     "lint:js": "eslint ./*.js app config lib server tests",
     "start": "ember serve --port=4200",
-    "test": "ember test"
+    "test": "ember test",
+    "postinstall": "node -e \"try { require('fs').symlinkSync(require('path').resolve('node_modules/@bower_components'), 'bower_components', 'junction') } catch (e) { }\""
```

Comment:
postscript and postinstall are different, The postinstall script here runs after every time you install a node package. What it does is that it symbolically links the `node_modules/@bower_components` directory to the `bower_components` directory. Hence whenever node runtime needs to access a file inside `node_modules/@bower_components`, it will redirect to the `bower_components` directory and access the file from there instead, Because specific packages are not available in yarn. Similar procedure is also being used in open-event